### PR TITLE
cleanup login command, removes need to visit /null

### PR DESCRIPTION
**Pull request by acourdavault**
## Description

Improve the login command:
- removes boilerplate of `beforeAll =()=>{cy.login2()}`
- removes the need to visit `/null` in each login
- renames functions more clearly

## Type of Change
- Non breaking change ('update', 'change')

### CODE REVIEW

File: packages/manager/cypress/support/login.js [hunk 1]
```diff
@@ -49,34 +48,52 @@ Cypress.Commands.add('login', () => {
...
 
-Cypress.Commands.add('login2', () => {
-  Cypress.on('uncaught:exception', (err, runnable) => {
-    // returning false here prevents Cypress from
-    // failing the test with newrelic errors
-    return false;
-  });
-  cy.visit('/null');
-  window.localStorage.setItem(
+const _loginWithToken = win => {
```
Comment (johnwcallahan) [inline packages/manager/cypress/support/login.js]: Can this be called `window`? Or if that interferes with global scope, `_window`? The abbreviation is kind of confusing IMO.

File: packages/manager/cypress/support/login.js [hunk 2]
```diff
@@ -49,34 +48,52 @@ Cypress.Commands.add('login', () => {
...
     'Bearer ' + Cypress.env('oauthtoken')
   );
-  window.localStorage.setItem(
-    'authentication/expire',
-    isoExpire
-  );
+  win.localStorage.setItem('authentication/expire', isoExpire);
+};
+
+Cypress.Commands.add('loginWithToken', () => {
```
Comment (johnwcallahan) [inline packages/manager/cypress/support/login.js]: Looks like this command isn't used anywhere; do we need it?

Comment (acourdavault) [reply packages/manager/cypress/support/login.js]: I agree it used to be useful, but it s probably not, and if needed someone could do `cy.visitWithLogin('/null')`. Will remove it

File: packages/manager/cypress/support/login.js (hunk 1 above)
Comment (acourdavault) [reply packages/manager/cypress/support/login.js]: Yes in theory the variable window exist in the browser, i used the pattern from the recipes in the doc:
https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/stubbing-spying__window-fetch/cypress/integration/stub-fetch-spec.js

**Comment by johnwcallahan** [comment]
I don't know what's going on with my machine, but these tests are so slow that I get failures unless I also increase the `defaultCommandTimeout`. Here are the results from three different runs,

## 1
Screen Shot 2020-03-23 at 9 34 49 AM

## 2
Screen Shot 2020-03-23 at 9 34 58 AM

## 3 (after increasing `defaultCommandTimeout` to 40000)
Screen Shot 2020-03-23 at 9 36 27 AM

**Comment by acourdavault** [comment]
Actually, I makes sense that defaultcommandtimeout must be increased too, these commands can take as long as needed for the page to load

**Pull request merged**